test_regress/t/t_order_dpi_export_3.cpp

```diff
@@ -0,0 +1,40 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+//
+// Copyright 2021 by Geza Lore. This program is free software; you can
+// redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+
+#include <svdpi.h>
+
+#include <Vt_order_dpi_export_3.h>
+#include <Vt_order_dpi_export_3__Dpi.h>
+
+void toggle_other_clk(svBit val) { set_other_clk(val); }
+
+void toggle_third_clk(svBit val) { set_third_clk(val); }
+
+int main(int argc, char* argv[]) {
+    Vt_order_dpi_export_3* const tb = new Vt_order_dpi_export_3;
+    tb->contextp()->debug(false);
```

Comment:
Better to pass 0 as takes an int, but even better to parse the standard command line so you can debug at runtime by passing the +verilated+debug+9 command line arg.